saleor/settings.py

```diff
@@ -473,9 +474,12 @@ def get_host():
 # Django GraphQL JWT settings
 GRAPHQL_JWT = {
     "JWT_PAYLOAD_HANDLER": "saleor.graphql.utils.create_jwt_payload",
+    # How long until a token expires, default is 5m from graphql_jwt.settings
+    "JWT_EXPIRATION_DELTA": timedelta(seconds=1),
```

Comment:
Why 1 second?

Reply:
Because I was testing and forgot about this 🤦‍♀️ Thanks for noticing!